src/main/webapp/app/exercises/shared/exam-exercise-row-buttons/exam-exercise-row-buttons.component.html

```diff
@@ -96,6 +96,7 @@
 
     <a
         *ngIf="course.isAtLeastInstructor && exercise.type !== exerciseType.QUIZ"
+        type="submit"
```

Comment:
This doesn't look correct, see https://github.com/ls1intum/Artemis/pull/2669#discussion_r568288094.

Reply:
```suggestion
```